sips/sip-026/sip-026-sbtc_peg.md

```diff
@@ -0,0 +1,171 @@
+- **Bitcoin’s Slow Transaction Times:** Bitcoin is limited in its ability to process large amounts of data quickly and efficiently. Today, Bitcoin creates new blocks every 10 minutes on average, an interval which is longer than some alternative blockchains. This prohibits many types of applications that require faster confirmation times. 
+
+## Proposed Solution
+sBTC aims to solve Bitcoin’s limitations by combining the capability of the Stacks Blockchain with the stability of Bitcoin’s value. By enabling secure movement of BTC in and out of the Stacks Blockchain via the sBTC protocol, users can interact with their BTC on Stacks using Clarity smart contracts and fast block times. Users can deposit BTC into the protocol, seamlessly transact using sBTC on the Stacks blockchain and have the freedom to redeem sBTC tokens for the underlying BTC at any time. 
+
+- **Programmability:** [Clarity](https://docs.stacks.co/clarity/overview) is the smart contract language on Stacks, which allows developers to encode essential business logic on a blockchain. Using smart contracts, developers can build more expressive decentralized applications that interact with sBTC, such as DeFi protocols, stablecoins, payments and more.
+- **Fast Blocks:** The Stacks Nakamoto Upgrade, proposed in [SIP-021](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md#proposed-solution), enables fast blocks where “user-submitted transactions will now take on the order of seconds, instead of tens of minutes.” Thus, sBTC on Stacks Nakamoto will offer an improvement to Bitcoin’s current transaction times. 
+
+The sBTC protocol not only addresses the limitations of the Bitcoin scripting system but also provides a secure and decentralized solution for utilizing Bitcoin in various applications.
+
+## Design
+This proposal describes a system in which the following are true:
+
+- sBTC is a SIP-10 token backed by 1:1 by BTC.
+- The sBTC peg wallet is maintained by the set of sBTC signers. These signers are responsible for the security and maintenance of the wallet, ensuring that sBTC is redeemable for BTC.
+- Bitcoin can be converted into sBTC within 3 Bitcoin blocks.
+- sBTC can be converted into Bitcoin within 6 Bitcoin blocks. This ensures sBTC on and off ramps are faster than other BTC assets on the market.
+- The sBTC SIP-10 token contract remains consistent across sBTC releases. This provides reliability for users and developers, meaning no adjustment from builders will be needed as the system evolves through subsequent sBTC releases.  
+
+## Overview of the sBTC Bootstrap Release
```

Comment:
This section and the next section strongly imply the existence of a separate sBTC implementation.  I don't think this SIP should talk about this at all, since we don't want to commit to that so openly.  Instead, this and the next section should instead simply enumerate all of the signer responsibilities that will be true regardless of what the sBTC implementation looks like.  Namely:

* Describing the responsibilities of the sBTC signers
   * They react to requests to deposit BTC
   * They react to requests to withdraw BTC
   * The stay online often enough that deposits and withdraws happen in a timely manner
   * They have to keep their signer hosts secure, so their keys don't get stolen
   * They have to proactively re-key every so often, which means moving the BTC to a new UTXO
* Signers must be independent entities 
* Signers must consolidate BTC as it is deposited
   * They must somehow ensure that that transaction fee is paid for (e.g. the user can specify the max fee, as you described in your design document)
* Signers must deduct transaction fees from users in order to fund BTC withdrawal transactions
   * They must somehow ensure that the transaction fee is paid for (e.g. they deduct it from the user, and they set a minimum sBTC withdrawal amount)
* Signers must coordinate to set and advertise the fee parameters of the system
   * They must decide a minimum sBTC peg-out
   * They must decide a STX transaction fee for minting the sBTC (who pays for this, btw?  If it's the signers, then this document must state that the signers must have a STX balance to pay it)

Once these are all listed, you should then use it to determine the criteria for selecting a suitable signer.